src/js/models/layer.js

```diff
@@ -283,6 +283,8 @@ define(function (require, exports, module) {
          */
         initialized: false,
```

Comment:
`Layer#initialized === true` indicates the Layer's following properties are initialized:

- essential properties
- bounds
- properties required as a selected layer
- extension data